### PR TITLE
Converts wheel storage from S3 to git-lfs

### PR DESCRIPTION
**Pull request by conorsch**
The binary wheels, and associated tarballs, previously stored in S3 buckets have now been moved into this repository, inside the `localwheels/` dir, via `git-lfs`. 

There's a significant drawback to this method: the public URL (https://pypi.securedrop.org/simple/) is only updated upon merge into the `master` branch of this repository. In order to test locally, e.g. during preparation of packaging changes, or during review of an unmerged PR, the docs now recommend using `python3 -m http.server` to spin up a local server, and editing the index URL in-place. That's a bit clunky, but with that small workflow change, we can ditch S3 entirely, in favor of the wonderfully more auditable git-lfs.

Closes https://github.com/freedomofpress/securedrop-debian-packaging/issues/109 

Practically speaking, we should review and merge https://github.com/freedomofpress/securedrop-debian-packaging/pull/112 first, before these changes, in order to reduce conflicts.

### Testing
Make sure you have git-lfs installed and configured!

Then, read through the documentation changes to make sure the new workflow is clear. Then try building a package. This is what I tried:

1. `PKG_VERSION=0.0.11 make securedrop-client`

That will pull from the public URL. Use the `python3 -m http.server` approach to test the local fetching, e.g. by bumping a pip dep and ensuring the new wheel can be found during package build.

Post-merge, we'll need to update the PyPI URL branch-tracking logic to track "master" (it's currently tracking this branch, to aid in review).

### CODE REVIEW

**Comment by conorsch** [comment]
Seeing an [odd CI failure](https://circleci.com/gh/freedomofpress/securedrop-debian-packaging/1989) _only_ on `build-stretch-securedrop-client`:

<blockquote>
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:720)'),)': /simple/alembic/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:720)'),)': /simple/alembic/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:720)'),)': /simple/alembic/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:720)'),)': /simple/alembic/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:720)'),)': /simple/alembic/
Could not fetch URL https://pypi.securedrop.org/simple/alembic/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.securedrop.org', port=443): Max retries exceeded with url: /simple/alembic/ (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:720)'),)) - skipping
</blockquote>

Since all the other jobs are passing, I suspect it's a transient error. Will let it alone for a while so it can think about what it's done, then restart CI and see if it straightens up and flies right.

**Comment by redshiftzero** [comment]
we discussed removing the stretch jobs during standup today and all were in agreement, so once #126 is merged you can rebase and then the offending CI job will be gone

**Comment by conorsch** [comment]
CI is passing, now that #112 & #126 are in. Ready for review.

File: .circleci/config.yml [hunk 1]
```diff
@@ -273,8 +272,8 @@ jobs:
       - image: circleci/python:3.7-buster
     steps:
       - checkout
+      - *installgitlfs
```
[the remark this answers is not in the document]
Comment (conorsch) [reply .circleci/config.yml]: Great call, we make the git-lfs recommendation in all the artifact repos, should absolutely do the same here. Will add!

[the remark this answers is not in the document]
Comment (conorsch) [reply .circleci/config.yml]: Do we just want to add the lfs to the `install-deps` action? Debian 10 has 2.7.1 in the dist repos, and the manual install logic in the CI config uses 2.7.2.

[the remark this answers is not in the document]
Comment (redshiftzero) [reply .circleci/config.yml]: yeah let's do that since debian has a packaged version

**Pull request merged**